db/migrate/20120903092956_devise_create_users.rb

```diff
@@ -34,7 +34,7 @@ def change
       # t.string :authentication_token
 
 
-      t.timestamps
+      t.timestamps null: true
```

Comment:
Went with what the deprecation message suggested. It's unclear if we will set created_at = Time.now && updated_at = nil.